eng/.docsettings.yml

```diff
@@ -58,7 +59,7 @@ known_content_issues:
   - ['sdk/keyvault/Microsoft.Azure.KeyVault/README.md','#5499']
   - ['sdk/servicebus/Microsoft.Azure.ServiceBus/README.md','#5499']
   - ['sdk/eventhub/Microsoft.Azure.EventHubs/README.md','#5499']
-  - ['sdk/eventhub/Azure.Messaging.EventHubs/README.md','#5499']
+  - ['sdk/alertsmanagement/Microsoft.Azure.Management.AlertsManagement/README.md','#5499']
```

Comment:
Ohh, could you explain to me what this is? I've never seen this file and I wonder if I have to do something similar to what you are doing for KV.

Reply:
This is the analysis ruleset that runs for documents at CI time.   I basically told it that it can start running against the new Event Hubs package and added a "please don't run against AlertsManagement" (which has been causing CI to fail for us all)

You may or may not want to tweak the KeyVault stuff if you can target just your new track two package in the directory structure.

Reply:
Awesome. Thank you!